modules/ROOT/pages/amazon/amazon-s3-connector-studio.adoc

```diff
@@ -39,13 +60,14 @@ image::amazon/amazon-s3-01.png[Global Element Properties]
 |Access Key |Alphanumeric text string that uniquely identifies the user who owns the AWS account.
 |Secret Key |Key that plays the role of a password.
 |Try Default AWS Credentials Provider Chain |Indicates whether to use temporary credentials.
+|Region Endpoint |URL entry point for a web service.
 |===
 +
 You can either enter your credentials into the global configuration properties or reference a configuration placed in a file in the `src` folder of your project file that contains these values. For simpler maintenance and better reuse of your project, Mule recommends that you use a configuration file.
 +
 . If you are:
 +
-* Using the default Amazon S3 storage, leave the default entries for *Proxy* and *Advanced* tabs.
+* Using the default Amazon S3 storage, leave the default entries for *Proxy*, *Security*, and *Advanced* tabs.
```

Comment:
. If you are using the default Amazon S3 storage, leave the default entries for *Proxy*, *Security*, and *Advanced* tabs.